requirements.txt

```diff
@@ -0,0 +1,7 @@
+lxml
+bleach==1.4.2
+oauthlib==1.0.3
+Mako
+git+https://github.com/edx/XBlock.git#egg=XBlock
```

Comment:
Is it the XBlock standard to require the XBlock depot using this unpinned form? Do other XBlocks do the same?